examples/src/Examples/UseLocalStorage.purs

```diff
@@ -32,17 +34,14 @@ view = withHooks do
       ]
     ]
 
-useLocalStorage :: HookName -> String -> String -> Hook { value :: String, setValue :: Dispatch String }
+useLocalStorage :: HookName -> String -> String -> Hook (Tuple String (Dispatch String))
 useLocalStorage (HookName name) key defaultValue = do
-  { state, setState } <- useState (HookName $ name <> ".State") defaultValue
+  state /\ setState <- useState (HookName $ name <> ".State") defaultValue
   useEffect (HookName $ name <> ".Effect") $ liftEffect do
     v <- getItem key =<< localStorage =<< window
     case v of
       Just v' -> setState v'
       Nothing -> setItem key defaultValue =<< localStorage =<< window
-  pure
-    { value: state
-    , setValue: \v -> do
-        setState v
-        setItem key v =<< localStorage =<< window
-    }
+  pure $ Tuple state \v -> do
```

Comment:
I think this should work with the operator too, right?

```suggestion
  pure $ state /\ \v -> do
```

Reply:
Yeah, I wasn’t sure which was more readable. I could even use it in the type signature if we want.